pkg/proxy/nftables/proxier.go

```diff
@@ -1745,6 +1761,31 @@ func (proxier *Proxier) syncProxyRules() {
 		}
 	}
 
+	// add interfaces to the flow table
+	// TODO: This should be all Pod interfaces using services and the external interfaces.
```

Comment:
We need to examine the scaling implications of this... The fact that the API forces you to explicitly list the interfaces you care about suggests that there's a per-interface cost of some sort (maybe even an n^2 cost on the total number of interfaces...).

Also, FWIW, as written, newly-created pods don't get to participate until after the next time a Service sync happens. This should be a separate reconciler, using netlink notifications.

Reply:
yes, this was a quick and dirty way of get this working

I think the rules in `setupNFTables` are ok, since those are never going to change, please ACK that

So, as you say here, this does not belong to this sync loop, so I was planning on creating another sync loop that just update the serviceFlowTable every time there is a conntrack event related to interfaces

My strawman approach will be
1. add a new option to specify a regex of interfaces to be accelerated
2. add a new loop like this
```go
       // passed as parameter
       r = regexp.MustCompile(kubeconfig.AcceleratedIntarfaceRegex)
      // subscribe to netlink events for interfaces
       nlChannel := make(chan netlink.LinkUpdate)
	doneCh := make(chan struct{})
	defer close(doneCh)
	if err := netlink.LinkSubscribe(nlChannel, doneCh); err != nil {
		klog.Infof("error subscring to netlink interfaces: %v", err)
	}
	ticker := time.NewTicker(1 * time.Minute)
	defer ticker.Stop()
        ifNames := []string{}
	for {
		ifaces, err := net.Interfaces()
		if err != nil {
			klog.Infof("error getting system interfaces: %v", err)
		}
                // skip down interfaces
                if f.Flags&net.FlagUp == 0 {
			continue
		}
                // skip loopback interfaces
		if f.Flags&net.FlagLoopback != 0 {
			continue
		}
               // skip interfaces that does not match the regex
               if !r.MatchString(f.Name) {
   			continue
		}
                ifNames = append(ifNames, f.Name)
           }

	if len(ifNames) > 0 {
		tx.Add(&knftables.Flowtable{
			Name:    serviceFlowTable,
			Devices: ifNames,
		})
	}
```